hal/armv7a/imx6ull/hal.c

```diff
@@ -134,33 +137,33 @@ static void hal_getMinOverlappedRange(addr_t start, addr_t end, mapent_t *entry,
 
 int hal_memoryGetNextEntry(addr_t start, addr_t end, mapent_t *entry)
 {
-	mapent_t tempEntry;
-	mapent_t minEntry;
+	int i;
+	mapent_t tempEntry = { 0 };
```

Comment:
It's not .bss

Reply:
Ok, changing it to draft - as not ready to be merged.

Reply:
In that case C used it's assignment (based on selected optimization) for `ia32` it's the worst case as I see:
![2022-03-18-161619_1638x383_scrot](https://user-images.githubusercontent.com/141153/159030346-d89d9d11-56dc-4aba-9de7-b624b19e0f3d.png)
On ARM it uses just two stores.